lib/ui/painting.dart

```diff
@@ -1174,13 +1174,31 @@ class Vertices extends NativeFieldWrapperClass2 {
       throw new ArgumentError("[indices] values must be valid indices in the positions list");
 
     Float32List encodedPositions = _encodePointList(positions);
-    Float32List encodedTexs = (textureCoordinates != null) ?
+    Float32List encodedTextureCoordinates = (textureCoordinates != null) ?
       _encodePointList(textureCoordinates) : null;
     Int32List encodedColors = (colors != null) ? _encodeColorList(colors) : null;
     Int32List encodedIndices = (indices != null) ? new Int32List.fromList(indices) : null;
 
     _constructor();
-    _init(mode.index, encodedPositions, encodedTexs, encodedColors, encodedIndices);
+    _init(mode.index, encodedPositions, encodedTextureCoordinates, encodedColors, encodedIndices);
+  }
+
+  Vertices.raw(
+    VertexMode mode,
+    Float32List positions, {
+    Float32List textureCoordinates,
+    Int32List colors,
+    Int32List indices,
+  }) {
+    if (textureCoordinates != null && textureCoordinates.length != positions.length)
+      throw new ArgumentError("[positions] and [textureCoordinates] lengths must match");
```

Comment:
Can you remove the square brackets, maybe replace them with double quotes? I think they come from an earlier time where we weren't sure about the style.

Also, periods at the end of the exception strings? ArgumentError doesn't say what the convention is but FlutterError says to be fully punctuated: https://docs.flutter.io/flutter/foundation/FlutterError/message.html